### ISSUE TITLE
Add an option to opt-out of sound effect

### ISSUE THREAD

**Issue by gergoradeczki** [kir-dev/sch-pincer#145]
Consider the following: you are commuting and left your volume at a considerable level. You order something. Your phone plays a dumb sounds. It's just bad UX.

**Comment by Gerviba** [comment]
This is an edge case because you can always use your volume control or mute the browser temporary. (And you should do it anyways if you're not using it with headphones!) Commuting with public transport the noise level is also high, so it won't affect your environment that much. We are talking about a 1.5sec sound file and you're use it once or twice a week. Your messenger notifications will cause more disturbance... 

We will add an option to disable sound effects to the profile menu.

**Comment by gergoradeczki** [comment]
No, we are talking about a "funny" feature that, sadly, became a permanent feature.

**Comment by Gerviba** [comment]
It was a planned permanent feature.
